content/en/docs/concepts/scheduling/configure-scheduler.md

```diff
@@ -0,0 +1,217 @@
+---
+title: How to Configure Your Kubernetes Scheduler
+reviewers:
+- wgliang
+content_template: templates/concept
+weight: 30
+toc_hide: false
+---
+
+{{% capture overview %}}
+
+This is third article of how to use and extend the Kubernetes scheduler series, the whole series of artices are,
+
+* [How does the Kubernetes scheduler work?](TODO)
+* [How to customize and extend the kubernetes scheduler?](/docs/concepts/scheduling/customization-and-extension/)
+* How to configure your kubernetes scheduler?
+* [Choose scheduling algorithms and strategies for your cluster](TODO)
+* [Learning kube-scheduler's design and implementation from source code.](TODO)
+
+This page explains how to configure Kubernetes scheduler.
+
+{{% /capture %}}
+
+{{% capture body %}}
+
+## Scheduler
+
+The Kubernetes scheduler is one of main components of Kubernetes and responsible for selecting the best node for the pod running against based on a series of predefined scheduler policies. The Kubenetes provides the default scheduler and policies to place the pod one by one to approprorite node in a Kubernetes cluster.
```

Comment:
Suggestion:
The [Kubernetes scheduler](docs/reference/command-line-tools-reference/kube-scheduler/) is a main component of Kubernetes. It is responsible for selecting the best node for the pod based on a series of predefined scheduler policies. Kubenetes provides the default scheduler and policies to place the pod on the appropriate node in a Kubernetes cluster.